### ISSUE TITLE
pronto-poper breaks GitHub and GitLab formatters

### ISSUE THREAD

**Issue by jeroenj** [mmozuras/pronto#78]
Since poper's message have no line number attached the uniqueness check breaks on both formatters (I did not check any other formatters, but text works).

It's basically because this line of code:

```ruby
messages = messages.uniq { |message| [message.msg, message.line.new_lineno] }
```

Because `message.line` is nil we get an undefined method exception for `new_lineno`.

**Comment by mmozuras** [comment]
Fixed (will be released with 0.5.0).

**Comment by jeroenj** [comment]
Great. Thanks! :+1:
